src/routes/GameScreen.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View } from 'react-primitives';
+import { ActivityIndicator } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { game, move } from '../services/GameService';
+import Grid from '../components/Grid';
+
+export default class GameScreen extends Component {
+    static navigationOptions = ({ navigation }) => ({
+        title: `Game #${navigation.state.params.game.id}`,
+    });
+
+    static propTypes = {
+        navigation: PropTypes.object.isRequired,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
Use instance field declaration here and remove the constructor. Like `static propTypes` but without `static`:

```js
static propTypes = {
    navigation: PropTypes.object.isRequired,
};

state = {
    ...
}
```